### PR TITLE
docs(FEC-8736): playlist

### PR DESCRIPTION
**Pull request by yairans**
### Description of the Changes

* playlist api jsdoc
* playlist guide 

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [x] Travis tests are passing (or test results are not worse than on master branch :))
- [x] Docs have been updated

### CODE REVIEW

File: flow-typed/types/playlist.js [hunk 1]
```diff
@@ -12,9 +12,9 @@ declare type KPPlaylistOptions = _KPPlaylistOptions;
 
 /**
  * @typedef {Object} KPPlaylistCountdownOptions
- * @param {number} [timeToShow] - When the countdown will appear (by default is towards the end)
- * @param {number} [duration=10] - How match time the countdown will appear
- * @param {boolean} [showing=true] - Whether to show the countdown
+ * @property {number} [timeToShow] - Shows when the countdown is scheduled to appear (by default, this is towards the end).
+ * @property {number} [duration=10] - Shows how the match time countdown will appear.
```
Comment (AnnaAleksandrowicz) [inline flow-typed/types/playlist.js]: Yair not sure what you meant by this line - can you clarify?

Comment (yairans) [reply flow-typed/types/playlist.js]: `duration` is a property which set how **much** time the countdown will displayed.  
is it clear now @AnnaAleksandrowicz ?

Comment (AnnaAleksandrowicz) [reply flow-typed/types/playlist.js]: Let's use "How long will the countdown appear" - OK?

**Comment by yairans** [comment]
where the mistake @AnnaAleksandrowicz ?

**Pull request merged**